pkg/printers/internalversion/printers.go

```diff
@@ -41,7 +41,6 @@ import (
 	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/util/sets"
-	"k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm"
```

Comment:
@luxas @sttts @justinsb `cmd/kubeadm/app/apis/kubeadm/well_known_labels.go` is not used/referenced. Shall we delete this `well_known_labels.go` file?